invenio_jobs/resources/resources.py

```diff
@@ -204,17 +204,10 @@ def read(self):
         return item.to_dict(), 200
 
     @request_view_args
-    @response_handler()
     def logs(self):
         """Read an item."""
-        identity = g.identity
-        hits = self.service.search(
-            identity=identity,
-            job_id=resource_requestctx.view_args["job_id"],
-            run_id=resource_requestctx.view_args["run_id"],
-            params=resource_requestctx.args,
-        )
-        return hits.to_dict(), 200
+        params = dict(q=resource_requestctx.view_args["run_id"])
+        return redirect(url_for("app-logs.search", **params))
```

Comment:
I am redirecting to keep the URL, other options would be to inject the app logs service to this resource as well (to be able to use the service method without redirecting), or if we can remove the endpoint, simply replace the link by the app logs endpoint (I would go with this one if we are okay with removing the endpoint), please share your thoughts.